tests/integration/integration.go

```diff
@@ -259,12 +264,12 @@ var _ = ginkgo.Describe("[Tx Processing]", ginkgo.Serial, func() {
 			require.Equal(1, instances[0].vm.Mempool().Len(context.Background()))
 		})
 
-		ginkgo.By("skip duplicate", func() {
+		ginkgo.By("skip mempool duplicate", func() {
 			_, err := instances[0].cli.SubmitTx(
 				context.Background(),
 				initialTx.Bytes(),
 			)
-			require.Error(err) //nolint:forbidigo
+			require.ErrorContains(err, vm.ErrNotAdded.Error()) //nolint:forbidigo
```

Comment:
We're getting the error from the rpc client, so we don't have the exact error type here, but we can still check that it includes the expected error string.